lel.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: cp437 -*-
+"┬ó┬ó┬ó┬ó┬ó┬ó" + f(4, x for x in range(1))
```

Comment:
Is adding this to the repository root intended?  What does it do?

Reply:
Is not intended